doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -128,6 +128,10 @@ Bluetooth samples
     * The front-end module test parameters are not set to their default value after the DTM reset command.
     * Added the vendor-specific ``FEM_DEFAULT_PARAMS_SET`` command for restoring the default front-end module parameters.
 
+  * :ref:`peripheral_hids_mouse` sample:
+
+    * Increased main stack size.
```

Comment:
Maybe mention the exact amount of stack? -> Increased the main stack size from x to y.

Reply:
Ok, changed to mention stack size